Vagrantfile

```diff
@@ -4,31 +4,27 @@
 require 'yaml'
 AWS = YAML.load_file 'aws.yml'
 
+if Vagrant::VERSION < '2.2.4'
+    puts "Vagrant version '" + Vagrant::VERSION.to_s + "' is not supported. Please upgrade to version 2.2.4 or higher"
+    exit
+end
+
 Vagrant.configure("2") do |config|
 
-    if Vagrant::Util::Platform.windows?
-        # needed for windows as prerequisite for vagrant-aws but will work from vagrant next release 2.2.4
-        if Vagrant::VERSION >= '2.2.4'
+    if ARGV[1] != 'dev' # aws plugin is needed only for non dev environment
+        if Vagrant::Util::Platform.windows?
+            # needed for windows as prerequisite for vagrant-aws
             required_plugins = [
             {"fog-ovirt" => {"version" => "1.0.1"}},
             "vagrant-aws"
             ]
             config.vagrant.plugins = required_plugins
```

Comment:
This line is duplicated. Move it after the if and use it once.

Reply:
done